backend/Tasque.Core.WebAPI/Controllers/BoardController.cs

```diff
@@ -1,31 +1,34 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
 using Tasque.Core.BLL.Services;
 using Tasque.Core.Common.DTO.Board;
-using Tasque.Core.Common.Entities;
-using Tasque.Core.Identity.Helpers;
 
 namespace Tasque.Core.WebAPI.Controllers
 {
     [Route("api/board")]
-    public class BoardController : EntityController<Board, BoardDto, BoardService>
+    [ApiController]
+    [Authorize]
+    public class BoardController : ControllerBase
     {
-        public BoardController(BoardService service, CurrentUserParameters currentUser)
-            : base(service, currentUser) { }
+        private readonly BoardService _boardService;
 
-        [Route("getUserBoards/{id}")]
-        [HttpGet]
-        public async virtual Task<IActionResult> GetUserBoards(int id)
+        public BoardController(BoardService service) 
         {
-            var boards = await _service.GetUserBoards(id);
-            if (boards is not null)
-            {
-                return Ok(boards);
-            }
-            else
-            {
-                return BadRequest("Entities not found");
-            }
+            _boardService = service;
         }
 
+        [HttpGet("{projectId}")]
+        public async Task<IActionResult> GetBoardByProjectId(int projectId)
+        {
+            var result = await _boardService.GetBoardByProjectId(projectId);
+            return Ok(result);
+        }
+
+        [HttpPut("{projectId}")]
```

Comment:
Do we need the project Id here, because you don't use it here?

Reply:
Changed